### PR TITLE
Update BYON-CallHandling-Service.yaml

### PR DESCRIPTION
**Pull request by JavierVillaLabarra**
Responding to the  [#issue24](https://github.com/camaraproject/WebRTC/issues/24) proposed by @deepakjaiswal1

We have added a  201 type response wich is returning a JSON object regarding the session parameters.

Feel free to add comments

@TEF-RicardoSerr
@pradeepachar-mavenir
@sushanthmavenir

### CODE REVIEW

**Comment by deepakjaiswal1** [comment]
@TEF-RicardoSerr @JavierVillaLabarra I believe it's unnecessary to include {status: 201, code: CREATED} in the responseBody for a 201 created response.

**Comment by deepakjaiswal1** [comment]
@JavierVillaLabarra I think we can omit the ObjectResponse schema. I would update Gereric201

FROM:
Generic201:
  description: Created
  content:
    application/json:
      schema:
        $ref: '#/components/schemas/ObjectResponse'
      examples:
        SessionCreatedMT183:
          $ref: '#/components/examples/exMT201_183'
        SessionCreatedMT200:
          $ref: '#/components/examples/exMT201_200'

TO:
Generic201:
      description: Created
      content:
        application/json:
          schema:
            $ref: '#/components/schemas/ReceiverSessionStatus'
          examples:
            SessionCreatedMT183:
              $ref: '#/components/examples/exMT201_183'
            SessionCreatedMT200:
              $ref: '#/components/examples/exMT201_200'

File: code/API_definitions/BYON-CallHandling-Service.yaml [hunk 1]
```diff
@@ -460,3 +523,14 @@ components:
             status: 503
             code: UNAVAILABLE
             message: Service unavailable
+    Generic201:
+      description: Created
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ReceiverSessionStatus'
+          examples:
```
Comment (pradeepachar-mavenir) [inline code/API_definitions/BYON-CallHandling-Service.yaml:532]: These examples are not relevant here.

Comment (pradeepachar-mavenir) [reply code/API_definitions/BYON-CallHandling-Service.yaml:532]: 201 Created is to convey to client that the request is accepted, and resource is created on WebRTC Gateway.  After 201, the gateway shall initiate the SIP transactions towards the network, based on transactions, subsequent notification shall be sent. 
My point was, listing exMT201_183 and exMT201_200 to be removed.

**Comment by TEF-RicardoSerr** [comment]
Hello @deepakjaiswal1 @pradeepachar-mavenir 
Just made the new commit, please check if it is aligned with your thoughts

**Review by pradeepachar-mavenir — APPROVED** [review]

**Pull request merged**